Mathlib/Data/Sign.lean

```diff
@@ -76,7 +76,7 @@ instance decidableEq : DecidableEq SignType := fun a b => by
 
 private lemma mul_comm : ∀ (a b : SignType), a * b = b * a := by rintro ⟨⟩ ⟨⟩ <;> rfl
 private lemma mul_assoc : ∀ (a b c : SignType), (a * b) * c = a * (b * c) := by
-  rintro ⟨⟩ ⟨⟩ ⟨⟩ <;> rfl
+  decide
```

Comment:
Do we have any idea what computation `decide` is doing here? The `rintro` proof is likely faster. If this were to be changed like this, then there ought be consistency with `mul_comm`.